appinventor/components/src/com/google/appinventor/components/runtime/Clock.java

```diff
@@ -207,6 +216,92 @@ public static Calendar MakeInstant(String from) {
     }
   }
 
+  /**
+   * An instant in time specified by number year, number month, number day
+   * @param year year integer
+   * @param month month integer
+   * @param day day integer
+   * @return  Calendar instant
+   */
+  @SimpleFunction(description = "Allows the user to set the clock to be "
+    +"a date value.\n" +
+    "Valid values for the month field are 1-12 and 1-31 for the day field.\n")
+  public Calendar MakeDate(int year, int month, int day) {
+    int jMonth = month - 1;
+    try {
+      GregorianCalendar cal = new GregorianCalendar(year, jMonth, day);
+      cal.setLenient(false);
+      cal.getTime();
+    } catch (java.lang.IllegalArgumentException e) {
```

Comment:
You shouldn't need to fully qualify IllegalArgumentException here because java.lang is always available.